scripts/velocity.py

```diff
@@ -0,0 +1,446 @@
+"""
+Generates reports on recently-closed issues on ZenHub and GitHub.
+"""
+import argparse
+import collections
+import datetime
+import email.utils
+import functools
+import getpass
+import json
+import logging
+import os
+import sys
+import time
+from typing import (
+    Mapping,
+    Union,
+)
+import urllib.parse
+
+import github
+import github.Issue
+import github.Requester
+import requests
+
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+
+DEFAULT_CONFIG_PATH = 'velocity.json'
+try:
+    DEFAULT_CONFIG_PATH = os.path.join(os.environ['project_root'],
+                                       DEFAULT_CONFIG_PATH)
```

Comment:
```suggestion
    DEFAULT_CONFIG_PATH = os.path.join(os.environ['project_root'], DEFAULT_CONFIG_PATH)
```
Maximum line length of 80 characters for documentation, code can go to 120 as long as it's not too complex of a statement.

Reply:
Okay. Is this still acceptable with the [updated contribution guidelines](https://github.com/DataBiosphere/azul/pull/2038/files#diff-ea1f65a8eaac0cec7ae151dfdbbbc587R25)?

Reply:
Ah sorry. I stand corrected.